tvb_build/deploy_with_keycloak/hbp_openshift/tvb_deployer.yml

```diff
@@ -32,33 +47,62 @@ objects:
       resources:
         requests:
           storage: "2Ti"
-  - kind: Pod
+  - apiVersion: v1
+    data:
+      .tvb.configuration: "ADMINISTRATOR_NAME=admin\r\nADMINISTRATOR_PASSWORD=1a1dc91c907325c69271ddf0c944bc72\r\nADMINISTRATOR_EMAIL=jira.tvb@gmail.com\r\nKEYCLOAK_CONFIGURATION=/home/tvb_user/keycloak_config/keycloak_web_config.json\r\nKEYCLOAK_WEB_CONFIGURATION=/home/tvb_user/keycloak_config/keycloak_web_config.json\r\nENABLE_KEYCLOAK_LOGIN=True\r\nTVB_STORAGE=/home/tvb_user/TVB_STORAGE/\r\nUSR_DISK_SPACE=5242880\r\nMATLAB_EXECUTABLE=/usr/bin/octave\r\nSELECTED_DB=postgres\r\nURL_VALUE=postgresql+psycopg2://tvb-postgres/${POSTGRESQL_DATABASE}?user=${POSTGRESQL_USER}&password=${POSTGRESQL_PASSWORD}\r\nWEB_SERVER_PORT=8080\r\nURL_WEB=http://127.0.0.1:8080/\r\nCLUSTER_SCHEDULER=oar\r\nMAXIMUM_NR_OF_THREADS=4\r\nMAXIMUM_NR_OF_OPS_IN_RANGE=2000\r\nMAXIMUM_NR_OF_VERTICES_ON_SURFACE=300000\r\nDEPLOY_CLUSTER=False\r\nLAST_CHECKED_FILE_VERSION=5\r\nLAST_CHECKED_CODE_VERSION=9339\r\nADMINISTRATOR_DISPLAY_NAME=Administrator"
+    kind: ConfigMap
+    metadata:
+      name: config-file
+  - kind: "DeploymentConfig"
     apiVersion: v1
     metadata:
       name: thevirtualbrain
-      labels:
-        app: tvb
     spec:
-      containers:
-        - env:
-          image: thevirtualbrain/tvb-run:keycloak
-          imagePullPolicy: Always
-          name: tvb-gui
-          ports:
-            - containerPort: 8080
-              protocol: TCP
-          volumeMounts:
-            - mountPath: /home/tvb_user/keycloak_config
-              name: keycloak-config
-            - mountPath: /home/tvb_user/TVB_STORAGE/
-              name: tvb-storage
-      volumes:
-          - name: keycloak-config
-            persistentVolumeClaim:
-              claimName: claim-keycloak-config
-          - name: tvb-storage
-            persistentVolumeClaim:
-              claimName: claim-tvb-storage
+      template:
+        metadata:
+          labels:
+            app: tvb
+        spec:
+          containers:
+            - name: tvb-gui
+              image: thevirtualbrain/tvb-run:keycloak
+              imagePullPolicy: Always
+              resources:
+                limits:
+                  cpu: '10'
+                requests:
+                  cpu: '1'
```

Comment:
is this no of cpu cores for one installation?
if it is, can we request 16 (N) for the web gui, and change the param MAXIMUM_NR_OF_THREADS accordingly (N-2)?

Reply:
This is the starting CPU no for a pod. Without launching another pod, the CPU no can increase automatically